### ISSUE TITLE
macOS login items - application icon broken.

### ISSUE THREAD

**Issue by kuehlriegecom** [microsoft/vscode#121795]
Issue Type: <b>Bug</b>

When you open the macOS Users & Groups, select the desired user account, switch the tab on the right side to "Login Items", scroll to Visual Studio Code.

VS Code version: Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d, 2021-04-13T09:36:32.643Z)
OS version: Darwin x64 20.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (8 x 2900)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 2, 2|
|Memory (System)|16.00GB (0.86GB free)|
|Process Argv|--crash-reporter-id 98855c9b-b8eb-40d3-9632-7869c18d4ba6|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
language-zpl|4O4|0.0.1
tcp-listener|4O4|0.0.1
zpl-testing-pack|4O4|0.0.1
zpl-viewer|4O4|0.0.1
vscode-edifact|DAX|0.7.0
xml|Dot|2.5.1
beautify|Hoo|1.5.0
code-zpl|Ian|0.1.0
vscode-styled-components|jpo|1.5.0
intellij-idea-keybindings|k--|1.4.4
rainbow-csv|mec|1.8.1
java|red|0.77.0
xslt-transform|Sve|0.0.4
sort-lines|Tyr|1.9.0
vscodeintellicode|Vis|1.2.12
vscode-java-debug|vsc|0.32.1
vscode-java-dependency|vsc|0.18.3
vscode-java-pack|vsc|0.13.0
markdown-all-in-one|yzh|3.4.0

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspyt653:30270858
pythonvspyt602cf:30291500
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vstry244:30276681
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
pythonvspyt639:30291489
pythontb:30283811
pythonvspyt551:30291414
vspre833:30267464
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30276952
pythondataviewer:30285071
vscus158:30286553

```

</details>

**Comment by kuehlriegecom** [comment]
![Screenshot 2021-04-21 at 09 18 50](https://user-images.githubusercontent.com/29857623/115514114-14022600-a284-11eb-9698-7aee678a37cc.png)

**Comment by kuehlriegecom** [comment]
![Screenshot 2021-04-21 at 09 44 26](https://user-images.githubusercontent.com/29857623/115520954-f5ebf400-a28a-11eb-891e-4c5226a02a99.png)
Same problem here..

**Comment by 9035768** [comment]
Adding to kuhriegecom's comments, it seems the broken icon applies to sizes below a size threshold.  For example, larger icons are not broken as shown in the image below.

![Inconsistency-Visual-Studio-Code-icons-after-update-to-1 55 2---larger-vs-smaller-sizes](https://user-images.githubusercontent.com/9868554/115922279-b2d87f00-a44a-11eb-80d6-755e116c0c1f.png)

**Comment by misolori** [comment]
[Please try clearing your icon cache](https://github.com/microsoft/vscode/issues/120935#issuecomment-816852205), this has been fixed in the 1.55.1 release.
